libs/inform/src/lib/abstracts/modal/modal.abstract.component.ts

```diff
@@ -27,4 +51,29 @@ export class NgxModalAbstractComponent<ActionType extends NgxModalActionType, Da
 	 * An emitter that will emit if the modal is closed
 	 */
 	@Output() public close: EventEmitter<void> = new EventEmitter<void>();
+
+	constructor(
+		@Inject(PLATFORM_ID) private platformId: string,
+		private readonly elementRef: ElementRef<HTMLElement>
+	) {}
+
+	public ngAfterViewInit(): void {
+		// Iben: If we are in the browser, check if either of the two accessibility labels are set
+		if (isPlatformBrowser(this.platformId) && (this.ariaLabelledBy || this.ariaDescribedBy)) {
```

Comment:
If neither the ariaLabelledBy nor the ariaDescribedBy property were passed, this will not throw an error. Is this by design?

Reply:
Yes, as there's also the option to pass an `aria-label`.